src/main/java/teammates/common/util/Utils.java

```diff
@@ -35,5 +37,23 @@ public static Gson getTeammatesGson() {
                 .disableHtmlEscaping()
                 .create();
     }
+    
+    /**
+     * Convert a list of {@link FeedbackParticipantType} to a string where each element will be separated by
+     * a delimiter.
+     */
+    public static String joinParticipantTypes(List<FeedbackParticipantType> participants, String delimiter) {
```

Comment:
There is a method `toString` in StringHelper.java that takes a list of string and concatenate them. I suggest that this can be turned to:
```
public static <T> String toString(List<T> list, String delimiter) {
        if (list.isEmpty()) {
            return "";
        }
        StringBuilder returnValue = new StringBuilder();
        for (int i = 0; i < list.size() - 1; i++) {
            returnValue.append(list.get(i)).append(delimiter);
        }
        //append the last item
        returnValue.append(list.get(list.size() - 1));
        
        return returnValue.toString();
    }
```

Which will work with all objects calling their own `toString` method. This will make the method reusable in all the code base.